.github/workflows/polkadot-companion-labels.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           authToken: ${{ secrets.GITHUB_TOKEN }}
           ref: ${{ github.event.pull_request.head.sha }}
-          contexts: 'continuous-integration/gitlab-check-polkadot-companion-build'
+          contexts: 'continuous-integration/check-dependent-polkadot'
```

Comment:
@s3krit is it allowed to put a list of jobs here? Substrate [depends](https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L582-L587) both on polkadot and cumulus companions

Reply:
As per the docs in the [readme](https://github.com/s3krit/await-status-action) yeah, you can provide a semicolon delimited list of contexts (i.e., check names). Never tried it in practice though.

Reply:
There is currently no label for `needscumuluspr` and I'm not that convinced to add this.